Library/ViewModels/RewardAddOnSelectionViewModel.swift

```diff
@@ -305,20 +263,19 @@ public final class RewardAddOnSelectionViewModel: RewardAddOnSelectionViewModelT
       refTag,
       configData,
       additionalPledgeAmount,
-      allRewardsShippingTotal,
       pledgeTotal
     )
     .take(first: 1)
     .observeForUI()
-    .observeValues { project, baseReward, selectedRewards, refTag, configData, additionalPledgeAmount, shippingTotal, pledgeTotal in
+    .observeValues { project, baseReward, selectedRewards, refTag, configData, additionalPledgeAmount, pledgeTotal in
       let checkoutPropertiesData = checkoutProperties(
         from: project,
         baseReward: baseReward,
         addOnRewards: selectedRewards,
         selectedQuantities: configData.selectedQuantities,
         additionalPledgeAmount: additionalPledgeAmount,
         pledgeTotal: pledgeTotal,
-        shippingTotal: shippingTotal,
+        shippingTotal: 0,
```

Comment:
Defaulting the shipping total to 0 so that everything can remain backward compatible. Eventually, the aim is to remove shipping from checkout API calls.